src/main/java/org/openjdk/jextract/impl/UnsupportedFilter.java

```diff
@@ -175,6 +175,14 @@ public Void visitTypedef(Typedef typedefTree, Declaration firstNamedParent) {
 
     @Override
     public Void visitConstant(Constant d, Declaration firstNamedParent) {
+        Type unsupportedType = firstUnsupportedType(d.type(), false);
+        String name = fieldName(firstNamedParent, d);
+        if (unsupportedType != null) {
+            warnSkip(name, STR."unsupported type usage: \{unsupportedType}");
+            Skip.with(d);
+            return null;
+        }
+
```

Comment:
Could you add a test case for this to ./test/testng/toolprovider/unsupported/TestUnsupportedTypes.java for this as well?

Reply:
Will add a test case.